GitFyle.Core.Api/Brokers/Storages/IStorageBroker.Repository.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using GitFyle.Core.Api.Models.Foundations.Repositories;
+using System.Threading.Tasks;
+
+namespace GitFyle.Core.Api.Brokers.Storages
+{
+    internal partial interface IStorageBroker
```

Comment:
This is the wrong file. I was expecting an addition on the existing `IStorageBroker.Repositories.cs`